zerver/openapi/zulip.yaml

```diff
@@ -12160,34 +12160,39 @@ paths:
                   $ref: "#/components/schemas/InviteRoleParameter"
                 stream_ids:
                   description: |
-                    A list containing the [IDs of the streams](/api/get-stream-id) that
-                    the newly created user will be automatically subscribed to if the
-                    invitation is accepted, in addition to default streams configured via
-                    the `include_realm_default_subscriptions` setting.
-
-                    This parameter must be the empty list if the current user has the
-                    unlikely configuration of being able to generate invitations while
-                    lacking permission to subscribe other users to streams.
+                    A list containing the [IDs of the channels](/api/get-stream-id) that the
+                    newly created user will be automatically subscribed to if the invitation
+                    is accepted, in addition to any default channels that the new user may
+                    be subscribed to based on the `include_realm_default_subscriptions`
+                    parameter.
 
-                    **Changes**: Prior to Zulip 9.0 (feature level 261), if the user was not
-                    allowed to subscribe others to stream, the newly created user was not
-                    subscribed to any streams.
```

Comment:
Just wanted to ask that this text has been remove intentionally? Although I agree that nothing has changed about how client should sent the request due to the change described above but still helps in conveying the change in result.

I do not have strong opinions about whether we should have this text or not, but just wanted to make sure that this has not been removed by mistake while making other changes.

Reply:
Thinking about this a bit, I think this isn't a change in the semantics of this parameter -- it's really just describing how `include_realm_default_subscriptions` would be imputed prior to this change -- namely as always False. I'll try drafting a "previously" entry for that.

Reply:
``` diff
diff --git a/zerver/openapi/zulip.yaml b/zerver/openapi/zulip.yaml
index d99716318f..7362ad668d 100644
--- a/zerver/openapi/zulip.yaml
+++ b/zerver/openapi/zulip.yaml
@@ -12187,7 +12187,10 @@ paths:
                     invitation does not generally have permission to [subscribe other
                     users to channels][can-subscribe-others].
 
-                    **Changes**: New in Zulip 9.0 (feature level 261).
+                    **Changes**: New in Zulip 9.0 (feature level 261). Previous versions
+                    of the Zulip behaved as though this parameter was always false;
+                    clients would copy the organization's default streams into the
+                    `stream_ids` parameter to request the default streams.
 
                     [default-channels]: /help/set-default-channels-for-new-users
                     [can-subscribe-others]: /help/configure-who-can-invite-to-channels
```